src/web/blocks.rs

```diff
@@ -28,19 +29,20 @@ pub struct BlockShallowDTO {
     timestamp: Timestamp,
     block_hash: SerScaleHex<Hash>,
     transactions: u32,
-    rejected_transactions: u32,
+    // rejected_transactions: u32,
```

Comment:
Could you find any data about rejected transactions in the current iroha data model? I mean, where this data is stored and how to retrieve it through Query API.

Reply:
Rejected Transaction are interleaved in ```iroha2-dev``` and are stored in ```iroha2-stable```.
So to proceed further would be to ignore rejected transactions.

Reply:
I am not sure that it should be handled this way in the explorer. In future, Explorer will be used most probably with a stable version of Iroha, and its frontend will always expect `rejected_transactions` field to be presented.

Therefore, I think that the solution would be to return `0` in case if Explorer works with Iroha which doesn't provide this data.